Readme.md

```diff
@@ -40,6 +42,7 @@ brew 'emacs', args: ['with-cocoa', 'with-gnutls']
 cask 'google-chrome'
 cask 'java' unless system '/usr/libexec/java_home --failfast'
 cask 'firefox', args: { appdir: '~/my-apps/Applications' }
+mac_app_store '1Password', 443987910
```

Comment:
Although, technically the name is only used in the UI and not passed to commands but makes it more readable.